action-scripts/build-config.js

```diff
@@ -1,8 +1,8 @@
 const fs = require("fs");
 
-const buildConfig = async ({
-  outputFolder = "build",
+const buildConfig = ({
   transformedTokenPath,
+  outputFolder = "build",
```

Comment:
Noticed that `outputFolder` and `transformedTokenPath` were listed in opposite order than where `buildConfig()` was called in `main.js`.

Reply:
This is good for readability, but shouldn't matter since the arguments are now destructured.